pkgs/by-name/fr/freac/package.nix

```diff
@@ -3,37 +3,198 @@
+  faac, # free = false
+  faad2,
+  fdk_aac,
+  ffmpeg,
+  flac,
+  lame,
+  libcdio,
+  libcdio-paranoia,
+  # libcdrip, # Missing from nixpkgs as of 2025/07/21 ?
+  libogg,
+  libopus,
+  libsamplerate,
+  libsndfile,
+  libvorbis,
+  # mac, # Same APE codec as monkeysAudio ?
+  monkeysAudio,
+  mp4v2,
+  mpg123,
+  # musepack, # Missing from nixpkgs as of 2025/07/21 ? or libmpcdec ?
+  rnnoise,
+  rubberband,
+  speex,
+  wavpack,
 }:
 
-stdenv.mkDerivation rec {
+# NOTE: For more details on building fre:ac for Linux & MacOS visit the following URLs:
+#   https://github.com/enzo1982/freac/blob/master/.github/workflows/tools/build-appimage
+#   https://github.com/enzo1982/freac/blob/master/.github/workflows/tools/build-macos
+#   https://github.com/enzo1982/freac/blob/master/tools/build-codecs
+
+# TODO: Add freac_latest to to make continuous build version available
+# TODO: Instead of copying external libs into the package derivation, can we instead link them to the package somehow?
+
+stdenv.mkDerivation (finalAttrs: {
   pname = "freac";
   version = "1.1.7";
 
   src = fetchFromGitHub {
     owner = "enzo1982";
     repo = "freac";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     sha256 = "sha256-bHoRxxhSM7ipRkiBG7hEa1Iw8Z3tOHQ/atngC/3X1a4=";
   };
 
-  buildInputs = [
-    boca
-    smooth
+  nativeBuildInputs = [
+    boca # Required for freac
+    smooth # Required for boca
     systemd
+
+    curl
+    gnome-icon-theme
+    openssl
   ];
 
+  buildInputs = [
+    # exhale # 1.2.0 - https://gitlab.com/ecodis/exhale
+    faac # 1.30 - https://github.com/knik0/faac
+    faad2 # 2.10.0 - https://github.com/knik0/faad2
+    fdk_aac # 2.0.3 - https://sourceforge.net/projects/opencore-amr/files/fdk-aac
+    ffmpeg # 7.1 - https://ffmpeg.org/releases
+    flac # 1.4.3 - https://ftp.osuosl.org/pub/xiph/releases/flac
+    lame # 3.100 - https://sourceforge.net/projects/lame/files/lame
+    libcdio # 2.1.0 - https://ftp.gnu.org/gnu/libcdio/
+    libcdio-paranoia # 10.2+2.0.1 - https://ftp.gnu.org/gnu/libcdio
+    # libcdrip # 2.4a - http://cdrip.org/releases/
+    libogg # 1.3.5 - https://ftp.osuosl.org/pub/xiph/releases/ogg
+    libopus # 1.5.2 - https://ftp.osuosl.org/pub/xiph/releases/opus
+    libsamplerate # 0.2.2 - https://github.com/libsndfile/libsamplerate
+    libsndfile # 1.2.2 - https://github.com/libsndfile/libsndfile
+    libvorbis # 1.3.7 - https://ftp.osuosl.org/pub/xiph/releases/vorbis
+    # mac # 10.82 - https://freac.org/patches
+    monkeysAudio # 10.82 - https://freac.org/patches
+    mp4v2 # 2.1.3 - https://github.com/enzo1982/mp4v2
+    mpg123 # 32.9 - https://mpg123.org/download
+    # musepack # 4.75 - https://files.musepack.net/source
+    rnnoise # 9acc1e5 - https://codeload.github.com/xiph/rnnoise
+    rubberband # 1.8.2 - https://breakfastquay.com/files/releases
+    speex # 1.2.1 - https://ftp.osuosl.org/pub/xiph/releases/speex
+    wavpack # 5.7.0 -https://www.wavpack.com/
```

Comment:
Why do we have the versions and links here? Our versions will soon no longer match and maintaining this sounds not very pleasant.